### ISSUE TITLE
Redirection issue http to https in Chrome/Firefox/IE

### ISSUE THREAD

**Issue by anoopabbu** [gophish/gophish#652]
Hi Team,

I have tried hard to get redirection done but its not happening.In Chrome/IE, when we type the url with http it start file downloading where in firefox it shows firfox inbuild error.Please update me for the same.

**Comment by jordan-wright** [comment]
We'll need more info to troubleshoot this.

What redirects do you mean? For phishing pages or admin pages? What have you tried specifically? What do you expect to happen?

**Comment by anoopabbu** [comment]
When user type url with http it would redirect to https. In Chrome/Firefox I am typing URL: https://127.0.0.1:3333/ Its working properly. When I remove s from from the URL and hit the URL http://127.0.0.1:3333/users. It starts file downloading in Chrome/ IE and in Firefox It show some symbol. I have tried all the way which I found on Git. I have write the code to change the redirection but I didn't got solution.
